cognite/replicator/time_series.py

```diff
@@ -91,6 +91,7 @@ def copy_ts(
     project_src: str,
     runtime: int,
     client: CogniteClient,
+    dst_ts: List[TimeSeries],
```

Comment:
Also need to update docstring here for this parameter as well

Reply:
Done